src/components/ui/ListForms.jsx

```diff
@@ -6,41 +6,31 @@ import { companies } from '../../redux/actions';
 
 class ListForms extends PureComponent {
   render() {
+    const yesOfCourse = this.props.questions.reduce((yes ,question) => {
+      if (question.questionSlug === 'recommendation') {
+        yes = question.answers[0]
+      }
+      return yes
+    }, 'croute')
+
     const panels = this.props.forms.map(form =>{
 
       let label;
 
-      switch(this.props.company) {
-        case companies.FASTIT:
-        case companies.THEODO_FR:
-          if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) >= 8 && form.recommendation === 'Oui bien sûr') {
-            if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) == 10){
-              label = <Label color="green" className="pinned">WOW!</Label>;
-            } else {
-              label = <Label color="olive" className="pinned">Success</Label>;
-            }
-          }
-          else {
-            label = <Label color="red" className="pinned">Red bucket</Label>;
-          }
-          break;
-        default:
-          if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) >= 8 && form.recommendation === 'Yes, absolutely') {
-            if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) == 10){
-              label = <Label color="green" className="pinned">WOW!</Label>;
-            } else {
-              label = <Label color="olive" className="pinned">Success</Label>;
-            }
-          }
-          else {
-            label = <Label color="red" className="pinned">Red bucket</Label>;
-          }
+      if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) >= 8 && form.recommendation === yesOfCourse) {
```

Comment:
I think the next step would be to compute that in the backend and add a new field to the form: `successType` that can be either: `wow`, `success` or `red-bucket` 😉